### DISCUSSION TITLE
Allow to re-build lightprobes data without baking LightmapGI again

### DISCUSSION THREAD

**Discussion by jcostello** [godotengine/godot-proposals#7188] [3D]
### Describe the project you are working on

3D Environment Lighting

### Describe the problem or limitation you are having in your project

Baking LightmapGI is time consuming. Re-Baking light probes without baking the LightmapGI would be helpful when manually placing lightprobes in the scene

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

A button light the LightmapBake but only for probes

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Read above

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No

### Is there a reason why this should be core and not an add-on in the asset library?

I don't think so

**Comment by AThousandShips** [comment]
Please provide some more detail to these two proposals, at least some idea how this is supposed to be implemented, otherwise open a discussion, as per the instructions:
> Proposal issues are required to explain in technical detail how the suggested change should be implemented. It is also preferred that the submitter of a proposal is ready to implement it if it was approved. If you have a more general idea for a feature but are not well versed in Godot's architecture or do not possess the necessary knowledge to implement it in the engine, feel free to open a discussion instead of an issue.

If you are unable we can convert it to a discussion for you

**Comment by jcostello** [comment]
Sorry, I was un aware of the discussions. Previously I believe here was the place to do so. Place convert it for me :)

**Comment by Calinou** [comment]
Superseded by https://github.com/godotengine/godot-proposals/discussions/7189.
